pkg/generator.go

```diff
@@ -423,14 +423,18 @@ func (g *Generator) generateImports(ctx context.Context) {
 func (g *Generator) GeneratePrologue(ctx context.Context, pkg string) {
 	g.populateImports(ctx)
 
-	if !g.config.Issue845Fix {
+	// If both `inpackage` and `outpkg` are set warn
+	if g.config.InPackage && g.config.Outpkg != "" {
```

Comment:
If I'm not mistaken, this deprecation warning will fire even if `Issue845Fix` is set to `true`.

Reply:
I'm sorry, you're completely right.